packages/pigeon/run_tests.sh

```diff
@@ -343,6 +390,7 @@ while getopts "t:l?h" opt; do
     flutter_unittests) should_run_flutter_unittests=true ;;
     ios_e2e_tests) should_run_ios_e2e_tests=true ;;
     ios_unittests) should_run_ios_unittests=true ;;
+    ios_swift_unittests) should_run_ios_swift_unittests=true ;;
```

Comment:
Can you add this to the `-l` output as well where the tests are listed please?

Reply:
Done